changelog/switch-skip-declaration.dd

```diff
@@ -0,0 +1,21 @@
+The deprecation for `switch` cases which skip the declaration of a variable has ended
+
+Usage of variable which is declared in another `switch` case now results in an error.
+
+---
+void test3(int i)
+{
+    switch (i)
+    {
+        case 1:
+        {
+            int j;
+        case 2:
+            ++j;
+            break;
+        }
+        default:
+            break;
+    }
+}
+---
```

Comment:
I think the example should identify where the error is, and provide corrective action.